src/Symfony/Component/Messenger/Tests/Fixtures/DummyMessageTyped.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Symfony\Component\Messenger\Tests\Fixtures;
+
+class DummyMessageTyped implements DummyMessageInterface
```

Comment:
I had to add a new object with typed property to assess the behavior of the test.

Can be removed with Symfony 6.0+